package.json

```diff
@@ -1,39 +1,44 @@
 {
-  "name": "goals-site",
+  "name": "goal-site",
   "version": "1.0.0",
   "private": true,
   "scripts": {
```

Comment:
Please fix the scripts as they are not running properly. 

Refer to this comment: https://github.com/Real-Dev-Squad/website-goals/pull/52#issuecomment-1221520333